plover/steno_dictionary.py

```diff
@@ -159,7 +161,11 @@ def reverse_lookup(self, value):
 
     def casereverse_lookup(self, value):
         return list(self.casereverse[value].keys())
-
+    
+    def regex_value_match(self, rpattern, match_limit=100):
```

Comment:
That particular limit is just there as sort of a sanity check in case the user searches for something insane like .*
I tried to set a maximum that would be more than enough to populate the lookup window in any given case. Can be moved into a constant.